akvo/rsr/views/project.py

```diff
@@ -45,6 +45,9 @@ def directory(request):
     """The project list view."""
     qs = remove_empty_querydict_items(request.GET)
 
+    # iati_status was renamed to sector in fa8094647b
+    qs['status'] = qs.pop('iati_status', [''])[0]
```

Comment:
This overwrites status with '' if there is no iati_status param. This is the case when applying the advanced filter on the project listing page.